src/action.ts

```diff
@@ -248,6 +248,14 @@ class Action {
       executionGraphReport = JSON.parse(fs.readFileSync(path.join(bundleDir, 'report.json')).toString())
     } catch (error) {
       core.warning(`Error downloading bundle files for execution graph ${executionGraphId}, error: ${error}`)
+    } 
+
+    try {
+      if (fs.existsSync(outputsDir)) {
+        this.rmdir(outputsDir)
+      }
+    } catch (error) {
+      core.warning(`Error removing the directory ${outputsDir}, error: ${error}`)
```

Comment:
Calling the function here will erase the artifacts before being pushed to GitHub, we have to call it after `uploadArtifacts`, passing `baseDir` from the `ActionResult` output of this function.

Reply:
Also, since you're already checking wether if the directory exists or not inside `rmdir`, you don't need to call `fs.existsSync` here again.